Dockerfile.build

```diff
@@ -15,7 +15,6 @@ RUN git config --system --add safe.directory '*'
 WORKDIR /workdir
 
 COPY pyproject.toml ./
-COPY poetry.lock ./
```

Comment:
Just for my understanding, why don't we use the lockfile here anymore?

Reply:
Ah yes, I forgot to add a line about that. I believe that it's not used when we do a `pip install .`. If my understanding of the specifics of poetry is correct, it's used when we do `poetry install` which installs the dev env, but when we do `pip install`, then the only thing that matters is the dependency specifications from pyproject.toml (https://github.com/py-cov-action/python-coverage-comment-action/blob/v3/pyproject.toml#L18-L23) which only specify wildcards. I thought it was a bit confusing to include an unused file.

But I can be wrong !

Reply:
Thanks for the explanation, that makes sense of course.